integration/testdata/jib/pom.xml

```diff
@@ -9,6 +9,7 @@
 
     <properties>
         <java.version>1.8</java.version>
+        <jib.maven-plugin-version>1.3.0</jib.maven-plugin-version>
```

Comment:
should this (and other instances of this) be defined in a parent?

Reply:
oh jeepers where would we define it?

Reply:
I don't know. Maybe don't do this. Just wondering.

Reply:
Could be worth doing in the integration tests but the examples should be standalone. I'll leave it for now: these values were hard-coded previously and making them properties makes testing easier.